inst/dummy_summary_report.html

```diff
@@ -0,0 +1,5 @@
+<ul>
```

Comment:
disappointed in your choice of ipsum :joy:

Reply:
Consider it gone!

Reply:
What is this using now? I just figured you might use a more esoteric placeholder text